en/components/drop_down.md

```diff
@@ -224,6 +224,38 @@ If the sample is configured properly, a list  of countries should be displayed a
     </button>
 </div>
 
+### igxDropDownItemNavigation directive
+
+To enable keyboard navigation for the igxDropDown component, the igxDropDownItemNavigation directive should be applied. It's recommended that the directive is applied to the element that is triggering the opening and closing the of the drop down. That element will also have the focus, and when using the directive it will handle all triggered events. By default the focus will always stay on the element that igxDropDownItemNavigation is applied on, and this happens because the igxDropDown doesn't focus its items by default.
```

Comment:
It's recommended that the directive is applied to the element that is triggering the opening and closing **the of the** drop down - this should be corrected.